myapp/.pylintrc

```diff
@@ -0,0 +1,593 @@
+[MASTER]
```

Comment:
do we need this file kamal or we can add it to ignore file?

Reply:
This is the configuration file for the pylint test, we can locate it in a different directory if you prefer it to be more hidden